helm/auto-oncall-chart/templates/certificate.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.Installation.V1.Autooncall.tls.enabled }}
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: auto-oncall
+  namespace: {{ .Values.namespace }}
+spec:
+  secretName: auto-oncall-certs
+  dnsNames:
+  - {{ .Values.Installation.V1.Autooncall.host }}
+  acme:
+    config:
+    - domains:
+      - {{ .Values.Installation.V1.Autooncall.host }}
+      http01:
+        ingress: ""
+  issuerRef:
+    kind: {{ .Values.Installation.V1.Autooncall.tls.issuertype }}
+    name: {{ .Values.Installation.V1.Autooncall.tls.issuername }}
+{{- end }}
```

Comment:
@giantswarm/team-batman: will we be able to still use templating for app-operator compatible charts?

Reply:
So you can use values from referenced configmap/secret for templating?